src/test/java/com/deloitte/ddwatch/services/QualityReportServiceTest.java

```diff
@@ -0,0 +1,46 @@
+package com.deloitte.ddwatch.services;
+
+import com.deloitte.ddwatch.model.QualityReport;
+import com.deloitte.ddwatch.model.SonarQubeReport;
+import com.deloitte.ddwatch.model.Status;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class QualityReportServiceTest {
```

Comment:
Add tests for **create(InputStream inputStream, QualityReport qualityReport)** as well.